src/main/java/biweekly/util/DateTimeComponents.java

```diff
@@ -300,7 +300,9 @@ public String toString() {
 	 * @return the date string
 	 */
 	public String toString(boolean includeTime, boolean extended) {
-		NumberFormat nf = new DecimalFormat("00");
+		NumberFormat nf = NumberFormat.getNumberInstance(Locale.ENGLISH);
+		nf.setMinimumIntegerDigits(2);
+                nf.setMaximumIntegerDigits(2);
```

Comment:
Thanks for the code submission.  Can you edit line 305 so it uses tabs instead of spaces for indentation?  Thanks.